### PR TITLE
testcase/network: fix svace issue

### PR DESCRIPTION
**Pull request by HONGCHAEHEE**
fix a svace issue of network testcases

### CODE REVIEW

File: apps/examples/testcase/le_tc/network/tc_net_dup.c [hunk 1]
```diff
@@ -107,6 +107,7 @@ static void tc_net_dup2_p(void)
 	int sock_new = socket(AF_INET, SOCK_DGRAM, 0);
 	if (sock_new == -1) {
 		UTC_NET_LOGE("get new socket error (%d)\n", errno);
+		return;
```
Comment (pillip8282) [inline apps/examples/testcase/le_tc/network/tc_net_dup.c]: calling close(socket_old) is neccessary.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/examples/testcase/le_tc/network/tc_net_dup.c (hunk 1 above)
Comment (HONGCHAEHEE) [reply apps/examples/testcase/le_tc/network/tc_net_dup.c]: ok

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/examples/testcase/le_tc/network/tc_net_ether.c [hunk 2]
```diff
@@ -60,22 +59,23 @@ static void tc_net_ether_ntoa_p(void)
 	ifr = ifcfg.ifc_req;
 
 	fd = socket(AF_INET, SOCK_DGRAM, 0);
-	TC_ASSERT_NEQ("ether_ntoa", fd, -1)
+	TC_ASSERT_NEQ_CLEANUP("ether_ntoa", fd, -1, close(fd))
```
Comment (pillip8282) [inline apps/examples/testcase/le_tc/network/tc_net_ether.c]: if 'fd' is -1 then we should not close(fd). please fix it

**Pull request merged**